code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -607,6 +613,11 @@ func (sc *StorageSmartContract) saveUpdatedAllocation(all *StorageNodes,
 		return
 	}
 
+	err = emitAddOrOverwriteAllocation(alloc, balances)
+	if err != nil {
+		return
```

Comment:
No error wrapping here?

Reply:
will add

Reply:
that's happening on it's parent function `updateAllocationRequestInternal`
just error return should be good.